api/src/modules/impact/dto/impact-table.dto.ts

```diff
@@ -161,6 +163,17 @@ export class GetRankedImpactTableDto extends BaseImpactTableDto {
   @IsUUID(4)
   scenarioId?: string;
 
+  @ApiPropertyOptional({
+    description:
+      'Optional property for groupBy region and chart view to choose level of admin regions to show',
+  })
+  @IsOptional()
+  @Type(() => Number)
+  @IsNumber()
+  @Min(1)
+  @Max(3)
```

Comment:
I think the minimum should be 0, as discussed. If a user wants to select 6 countries and see a 4 ranked still at country level, that depth should equal 0

Also, I wouldn't set a fixed max limit. Even tho the current limit makes sense, I think its a better idea to not set a max limit and just uncollapse the tree as much as it can be uncollapsed.

i.e, users sends a 5 value but we have a maximum depth of 3, just send back a 3 level deep tree